tests/integrations/server/gc_worker.rs

```diff
@@ -66,3 +66,59 @@ fn test_gc_bypass_raft() {
         }
     }
 }
+
+// Raftstore-v2 introduces `Tablet` to manage regions, encapsulating the
+// operations on KVDB. This case ensures it's performed correctly.
+#[test]
+fn test_gc_bypass_raft_v2() {
+    let (cluster, leader, ctx) = test_raftstore_v2::must_new_cluster_mul(2);
```

Comment:
why not reuse test_gc_bypass_raft?

Reply:
Cannot do this directly. 
The way of getting `get_engine()` in `raftstore-v2` is different from `raftstore`

Reply:
Refactored and removed all redundant parts after reviewing the methods supplied by `Cluster`.